PMMain/Midas/MidasGUI/MouseCtrl.cpp

```diff
@@ -31,14 +31,73 @@ void MouseCtrl::setScrollRate(int rate)
 
 void MouseCtrl::setMinMoveXTimeDelta(unsigned int rate)
```

Comment:
All of the commented-out code is in progress? But the rest of the code is still functional, right?